### DISCUSSION TITLE
Having trouble removing padding from Expander

### DISCUSSION THREAD

**Discussion by ajmcateer** [AvaloniaUI/Avalonia#9232] [Q&A] [answered]
I am trying to remove the padding around the expander content or at least make it the same color as the center of the content. Not that I think about that might be preferred as I like the spacing currently just not the ugly square donut.

![image](https://user-images.githubusercontent.com/7401629/196550257-e867c4ae-7304-4986-bc1f-5af983f2c29d.png)

```
<Window.Styles>
  <Style Selector="Expander:expanded /template/ Border#ExpanderContent">
	  <Setter Property="Padding" Value="0"/>
  </Style>
  <Style Selector="Expander:expanded /template/ Border#ExpanderContent">
	  <Setter Property="Panel" Value="#202020"/>
  </Style>
</Window.Styles>
```

I had a short discussion here and I tried what he said but it still isn't working

https://github.com/AvaloniaUI/Avalonia/discussions/7733#discussioncomment-3798150

**Comment by robloo** [comment, marked as answer]
I think we need to know which version of Avalonia you are using. Priority of property setters was changed and this sort of thing might be affected. That said, why don't you use the light-weight styling resource directly?

https://github.com/AvaloniaUI/Avalonia/blob/f7f17224bb5684136cbd62a9871d73d3db03f40a/src/Avalonia.Themes.Fluent/Controls/Expander.xaml#L48

Just re-define that in the resources of your window or the control directly to what you would like and it should just work -- no need for styles.

Side note: Expander padding by default applies to the header area NOT the content area. I fixed that in upstream WinUI a long time ago but it appears Avalonia was never updated. I guess I should add that to my todo list before 11.0.

**Comment by Splitwirez** [comment]
> square donut

![image](https://user-images.githubusercontent.com/34009058/198148223-03f762ef-934f-4b8a-a845-20ad48a80918.png)
